### DISCUSSION TITLE
Turning Off Previews

### DISCUSSION THREAD

**Discussion by fuzenco** [Feel-ix-343/markdown-oxide#128] [Q&A] [answered]
I’ve got this installed in Zed but the previews shown when hovering over text/headings is so distracting. Is there a way to turn that off?

**Comment by Feel-ix-343** [comment, marked as answer]
Hey! Thank you for the comment.

There is not currently a way to do this, but we certainly could.

Here we go #129

**Reply by fuzenco** [reply]
This will be great when released 👍🏼.

**Reply by Feel-ix-343** [reply]
check it out on main

**Reply by Feel-ix-343** [reply]
add `hover = false` to .config/moxide/settings.toml

**Reply by fuzenco** [reply]
I’d not seen your reply earlier. Many thanks, @Feel-ix-343.
